### DISCUSSION TITLE
已经付钱了，但是还是显示免费版，不知道怎么弄的。

### DISCUSSION THREAD

**Discussion by raysir66** [getlantern/lantern-forum-cn#252] [Q&A]
邮箱 dengrui66@foxmail.com

![41710740924_ pic](https://github.com/getlantern/lantern-forum-cn/assets/64630747/409d6995-dc87-4dcf-a1dc-f3d2a68a6a39)
![51710741018_ pic](https://github.com/getlantern/lantern-forum-cn/assets/64630747/0e146a52-fda2-4f6c-a0a4-3dbb4210596a)
![61710741047_ pic](https://github.com/getlantern/lantern-forum-cn/assets/64630747/ee3e1ac2-5233-47db-a017-55bd2ea8e4a0)

**Comment by MaxMKMax** [comment]
@raysir66 您好。我可以知道您使用什么设备吗？您应该会收到激活码来登录专业版。请检查您的电子邮件以获取激活码。如果不起作用，我会帮助您通知付款，为您手动升级。

我会帮您转达给我们的负责人。他们将于今晚按照美国时区上线。会尽快帮您解决。抱歉让您久等了。谢谢!

**Reply by raysir66** [reply]
您好，这边使用的是macbook pro。邮箱里没有看到激活码，垃圾邮箱也检查过。如果能帮手动升级，最好了
![WechatIMG8](https://github.com/getlantern/lantern-forum-cn/assets/64630747/ee109f71-267a-4632-b2b1-04f3cfcb3a7b)

**Reply by MaxMKMax** [reply]
您好, Okay! 我帮您提交您的问题给支付团队。他们会手动帮助您升级到专业版。非常抱歉影响了您的体验感, 谢谢您对蓝灯的支持。

**Reply by raysir66** [reply]
@Mike777Khoo 您好，请问可以了吗，暂时看到的还是免费版。另外咨询下，没有看到有登录的地方，能用邮箱登录vpn吗

**Reply by MaxMKMax** [reply]
@raysir66 我很抱歉地通知您，当前的 IOS/Mac 无法登录专业版。但我们很快就会发布新版本。请稍等我们一下。感谢您的等待。

如果您是受到授权码问题影响的ios用户，我们将通过重新延长您的专业版使用时间来补偿您。

**Reply by raysir66** [reply]
大概什么时候发布呢。

**Reply by MaxMKMax** [reply]
@raysir66 
我也不清楚呢. 但应该在一两周里面把. 明晚我会和他们开会, 我来帮您问一下吧. 看能不能追一下他们!

**Reply by raysir66** [reply]
怎么样了。太久了的话，帮退款吧。等支持了，我再重新买吧

**Reply by MaxMKMax** [reply]
@raysir66 我们这周发布了新的 Flutter 桌面应用程序测试版（8.0.0 版本）。

Mac - https://github.com/getlantern/lantern-binaries/raw/main/lantern-installer-preview.dmg

Windows - https://github.com/getlantern/lantern-binaries/raw/main/lantern-installer-preview.exe

Linux - https://github.com/getlantern/lantern-binaries/raw/main/lantern-installer-preview-64-bit.deb

**Reply by MaxMKMax** [reply]
@raysir66 
您的设备有这些吗? 最新版本都能正常使用了. 但iOS还需要等. 你不介意的话之后我们帮您免费延长专业版来当作补偿. 这样就不需要退款了. 发布IOS了我会马上通知你.

**Reply by raysir66** [reply]
更新了最新的mac版。请问是在这里登录吗。可是我没有收到什么激活码。应该在哪里看到呢
![WX20240411-095731@2x](https://github.com/getlantern/lantern-forum-cn/assets/64630747/47beac9a-09c6-409e-95f7-0a828171c7b2)

**Reply by MaxMKMax** [reply]
@raysir66 您好. 一般您会收到邮件的. 在垃圾邮箱查看.

**Comment by MaxMKMax** [comment]
@raysir66 我也顺便帮您问付款团队了.

**Reply by raysir66** [reply]
垃圾箱也没有看见，还是帮忙推吧，谢谢

**Reply by MaxMKMax** [reply]
@raysir66 激活码: FG66X-BGQG4-WT668-48DGF-WQGHF

**Comment by raysir66** [comment]
激活成功了，谢谢您 @Mike777Khoo
